### PR TITLE
Fix broken links on Using a KMS provider for data encryption page

### PR DESCRIPTION
**Pull request by kadtendulkar**
Follow up on #35449

Updated page [Using a KMS provider for data encryption](https://kubernetes.io/docs/tasks/administer-cluster/kms-provider)

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kms-provider.md [hunk 1]
```diff
@@ -57,11 +57,11 @@ You can develop a KMS plugin gRPC server using a stub file available for Go. For
 you use a proto file to create a stub file that you can use to develop the gRPC server code.
 
 * Using Go: Use the functions and data structures in the stub file:
-  [service.pb.go](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1/service.pb.go)
+  [service.pb.go](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1/api.pb.go)
```
Comment (mk46) [inline content/en/docs/tasks/administer-cluster/kms-provider.md:60]:
```suggestion
  [api.pb.go](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1/api.pb.go)
```

File: content/en/docs/tasks/administer-cluster/kms-provider.md [hunk 2]
```diff
@@ -57,11 +57,11 @@ You can develop a KMS plugin gRPC server using a stub file available for Go. For
...
   to develop the gRPC server code 
 
 * Using languages other than Go: Use the protoc compiler with the proto file:
-  [service.proto](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1/service.proto)
+  [service.proto](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1/api.proto)
```
Comment (mk46) [inline content/en/docs/tasks/administer-cluster/kms-provider.md:64]:
```suggestion
  [api.proto](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1/api.proto)
```

**Review by mk46 — COMMENTED** [review]
few nits. otherwise LGTM

**Comment by mk46** [comment]
Thanks!!
/label tide/merge-method-merge
/lgtm

**Comment by divya-mohan0209** [comment]
@kadtendulkar Please note the comments on the issue from Tim that have specific requests for links other than the one to the master branch since it is NOT stable.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Comment by kadtendulkar** [comment]
Hello @divya-mohan0209 please approve PR

**Pull request merged**